### PR TITLE
feat: update integrations docs

### PR DESCRIPTION
**Pull request by DiRaiks**
Update:
- SDK page (add new SDK links)
- API page (add Holesky links)
- Subgraph (change URL to new one after moving to L2)
- Wallets (remove embedding example)

### CODE REVIEW

File: docs/integrations/sdk.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ React components for Lido Finance projects.
...
 
-Library for interaction with Lido contracts.
+Library for interaction with Lido protocol.
 
-- GitHub: [https://github.com/lidofinance/lido-js-sdk](https://github.com/lidofinance/lido-js-sdk)
-- NPM: [https://www.npmjs.com/search?q=%40lido-sdk/](https://www.npmjs.com/search?q=%40lido-sdk/)
+- GitHub: [https://github.com/lidofinance/lido-ethereum-sdk](https://github.com/lidofinance/lido-ethereum-sdk)
+- NPM: [https://www.npmjs.com/package/@lidofinance/lido-ethereum-sdk](https://www.npmjs.com/package/@lidofinance/lido-ethereum-sdk)
 
 ### Lido Python SDK
```
Comment (TheDZhon) [inline docs/integrations/sdk.md:20]: Shall we deprecate the Python SDK?

**Review by TheDZhon — APPROVED** [review]
LGTM :+1:

File: docs/integrations/sdk.md (hunk 1 above)
Comment (TheDZhon) [reply docs/integrations/sdk.md:20]: The better option might be to explicitly outline the target audience for each SDK/UI kit, wdyt, @DiRaiks?

Comment (DiRaiks) [reply docs/integrations/sdk.md:24]: @TheDZhon, good suggestion!
I added `Target audience` for each sdk description

**Review by TheDZhon — APPROVED** [review]
:rocket:

**Pull request merged**